analyses/tcd/notebooks/forecasts/tcd_iri_seas_forecast_probability.md

```diff
@@ -121,8 +121,8 @@ df_bavg["meet_thresh"] = np.where(df_bavg.prob >= thresh, True, False)
 ```
 
 ```python
-#get sum of leadtimes that predict above the threshold
-#for the given predicted period and coordinates
+# get sum of leadtimes that predict above the threshold
```

Comment:
I don't follow why you sum lead times?

Reply:
It's to calculate the percentiles, getting total number within each lead-time that met threshold, then dividing by total number.